init-containers/hamm-cost-updater/hamm-cost-updater/app.env.ctmpl

```diff
@@ -0,0 +1,5 @@
+# Removed the new relic javaagent
+export JAVA_OPTS="-server -Xms2g -Xmx2g -XX:NewSize=512m -XX:MaxNewSize=512m -XX:SurvivorRatio=8 -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:+CMSParallelRemarkEnabled -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=60 -XX:+ScavengeBeforeFullGC -XX:+CMSScavengeBeforeRemark -XX:+PrintGCDateStamps -verbose:gc -XX:+PrintGCDetails -XX:+PrintClassHistogramBeforeFullGC -XX:+PrintClassHistogramAfterFullGC -Dconfig.file=/etc/application.conf -Dcom.sun.jndi.ldap.connect.pool.protocol=plain -Dcom.sun.jndi.ldap.connect.pool.maxsize=100"
+export SLEEP=180
+export GOOGLE_OAUTH_CLIENT_ID=806222273987-j8gk6rft1e8vdb0s9f97stfps4m42491.apps.googleusercontent.com
```

Comment:
what is this client id and should it vary by env?

Reply:
It is the oauth client ID. There is one per app per environment. The oauth ID in google needs to include URLs of everywhere google will be asked to redirect the oauth response to. So we may not need a new client ID for each deployment (we probably won't be able to provision that many) but we do probably need to think about how we're going to keep everything in line.